src/languages/php/xdebug.md

```diff
@@ -0,0 +1,82 @@
+# Using Xdebug
+
+[Xdebug](https://xdebug.org/) is a real-time debugger extension for PHP.  While usually used for local development, it can also be helpful for debugging aberrant behavior on the server.  It is available on Platform.sh Grid instances running PHP 7.2 and higher.
+
+As configured on Platform.sh, it avoids any runtime overhead for non-debug requests, even in production, and only allows SSH-tunneled connections to avoid any security issues.
+
+## Setting up Xdebug
+
+Xdebug is not enabled the same way as other extensions, as it should not be active on most requests.  Xdebug has a substantial impact on performance and should not be run in production.
+
+Instead, Xdebug can be enabled by adding the following configuration to the application's `.platform.app.yaml` file:
+
+```yaml
+runtime:
+    xdebug:
+        idekey: PHPSTORM
```

Comment:
This should probably be a secret, more complex key, for production?

Reply:
In practice, I'm not sure it means anything.  Security is handled through SSH, not the idekey.  The IDE key is not a password-like field.